### ISSUE TITLE
The training results on the Dancetrack did not achieve the results in the paper

### ISSUE THREAD

**Issue by yyyzjing** [Xavier-Lin/TrackSSM#4]
Hello, I benefited a lot from your excellent work, thank you very much!
The pt file you provided can achieve the effect in the paper, but when I trained by mysel, it didn't work. **I used a 4090 to train dancetrack, according to the paper's settings for 120 epochs, and the other parameters were all set in accordance with the dancetrack.yaml settings, but the results submitted to codalab only had a result of HOTA: 53 or so.** I would like to ask where the problem is?
Is it a problem with the training equipment?
However, when I was training dancetrack, the loss of Time_info_decoder was still written as loss = loss_l1.mean() + loss_giou.mean(). Shouldn't it only be loss = loss_l1.mean() here?
Looking forward to your reply!!!!

**Comment by Wangzi06** [comment]
@yyyzjing did you find a solution to this problem
